src/bin/driver.rs

```diff
@@ -487,6 +494,38 @@ mod app {
 
         let target = new_settings.stream_target.into();
         c.shared.network.lock(|net| net.direct_stream(target));
+
+        c.shared.output_state.lock(|output| {
+            for (i, (&new_output_enabled, old_output_enabled)) in new_settings
+                .output_enabled
+                .iter()
+                .zip(old_settings.output_enabled)
+                .enumerate()
+            {
+                if new_output_enabled && !old_output_enabled {
+                    if let Ok(Some(delay)) = output[i].enable().map_err(|e| {
+                        log::error!("Cannot enable output {:?}! {:?}", i, e)
+                    }) {
+                        handle_output_event::spawn_after(
+                            delay.convert(),
+                            i.try_into().unwrap(),
+                        )
+                        .unwrap();
+                    }
+                }
+                if !new_output_enabled && old_output_enabled {
+                    if let Ok(Some(delay)) = output[i].disable().map_err(|e| {
+                        log::error!("Cannot disable output {:?}! {:?}", i, e)
+                    }) {
+                        handle_output_event::spawn_after(
+                            delay.convert(),
+                            i.try_into().unwrap(),
+                        )
+                        .unwrap();
+                    }
+                }
```

Comment:
I think this could be a bit shorter if the API was `output[i].set_enable(new_output_enabled: bool)` and then `if new_output_enabled != old_output_enabled { ... }`. But not critical. Can also remain like this.